mkdocs.yml

```diff
@@ -60,6 +60,7 @@ nav:
               - Scripting (Beginner): tutorials/lua_basics_helloworld.md
               - Scripting (Intermediate): tutorials/lua_basics_lightbulb.md
               - Scripting (Advanced): tutorials/lua_basics_manticoin.md
+              - Scripting Race Timer: tutorials/race_timer.md
```

Comment:
should we call it `Scripting (Expert)`?

Reply:
Yeah, I wasn't sure on this one. If we follow the pattern of the previous ones, then yes, that sounds good.

If we do one that's even more advanced than "Expert", then we might need to come up with something else 😛